### PR TITLE
WIP: Update SRM, SCI, DSR and DSRP

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
test Ubuntu16.04 Release_default Build please

**Comment by tmat** [comment]
test Ubuntu16.04 Release_default Build please

**Comment by KevinRansom** [comment]
@dotnet-bot test Ubuntu16.04 Release_default Build please

File: src/buildfromsource/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj [hunk 1]
```diff
@@ -641,6 +641,25 @@
     <PackageReference Include="Microsoft.DiaSymReader.PortablePdb" Version="$(MicrosoftDiaSymReaderPortablePdbPackageVersion)" />
     <PackageReference Include="Microsoft.DiaSymReader" Version="$(MicrosoftDiaSymReaderPackageVersion)" />
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTuplePackageVersion)" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.5.0-preview2-26401-03"></PackageReference>
```
Comment (brettfo) [inline src/buildfromsource/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj]: These merge conflicts need to be resolved.  The version numbers were moved to `src/buildfromsource/targets/PackageVersions.props`.

This comment holds for all changes under `src/buildfromsource`.

If you want to disable dependency uptake for this then also comment out the bottom line of that `PackageVersions.props`.

**Comment by cartermp** [comment]
@tmat should this be closed?

**Pull request closed without merging**